lineup.js

```diff
@@ -1,9 +1,14 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((total, lineup) => {
```

Comment:
Small naming not, calculateTotalSalary is receiving a list and calling it lineup. Reduce then takes each object from the list and passes them, one at a time to the inner function. Here you have named that single object lineup as well which is going to be super confusing to another dev reading this. Something like `player` would be a better name since each object has information on individual players and the lineup is a list of them.